HW0/P4/P4.py

```diff
@@ -23,55 +23,112 @@
 
     # Create 3D axes for plotting
     ax = Axes3D(plt.figure())
-
+    
     #####################
     # Part 1:
     #
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
     #####################
+    
+    traj=np.loadtxt("P4_trajectory.txt", delimiter=",");
+    np.reshape(traj,(120,6));
+    traj=np.transpose(traj)
+    x_coords,y_coords,z_coords=traj[0:3,:];
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '--b', label='True trajectory')
+    ax.plot(x_coords, y_coords, z_coords,
+            '--b', label='True trajectory')
 
     #####################
     # Part 2:
     #
     # Read the observation array and plot it (Part 2)
     #####################
-
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
-
+    
+    meas=np.loadtxt("P4_measurements.txt", delimiter=",");
+    np.size(meas)
```

Comment:
Stupid question: doesn't this line just print the number of elements in the array, and not actually do anything "in the pipeline"?